src/coreclr/build.sh

```diff
@@ -386,6 +386,7 @@ handle_arguments_local() {
 }
 
 echo "Commencing CoreCLR Repo build"
+echo "WARNING: This build script is deprecated and will be deleted soon. Use the root build script to build CoreCLR. If you want to build the CoreCLR runtime without using MSBuild, use the build-native.sh script."
```

Comment:
Please file an issue that tracks the deletion of the sub-directory build scripts and add a link above this line. This way we broadcast our intent here and give contributors a place to discuss the issue.